### PR TITLE
TUP-17229:context value be overwrited after retrieve schema

### PR DESCRIPTION
**Pull request by wchen-talend**

### CODE REVIEW

**Comment by nrousseau** [comment]
where are the junits...?

**Comment by wchen-talend** [comment]
Need to use mock and power mock to do the test . So I didn't do the junit .

File: main/plugins/org.talend.metadata.managment.ui/src/main/java/org/talend/metadata/managment/ui/model/ProjectNodeHelper.java [hunk 1]
```diff
@@ -528,14 +528,11 @@ public static void fillCatalogAndSchemas(IMetadataConnection iMetadataConnection
...
-            if (((DatabaseConnection) iMetadataConnection.getCurrentConnection()).getUiSchema() != null) {
+            if (!temConnection.isContextMode()
+                    && ((DatabaseConnection) iMetadataConnection.getCurrentConnection()).getUiSchema() != null) {
                 temConnection.setUiSchema(((DatabaseConnection) iMetadataConnection.getCurrentConnection()).getUiSchema());
             }
 
-            if (((DatabaseConnection) iMetadataConnection.getCurrentConnection()).getSID() != null) {
-                temConnection.setSID(((DatabaseConnection) iMetadataConnection.getCurrentConnection()).getSID());
-            }
-
```
Comment (hcyi) [inline main/plugins/org.talend.metadata.managment.ui/src/main/java/org/talend/metadata/managment/ui/model/ProjectNodeHelper.java]: no need set the SID ?

Comment (wchen-talend) [inline main/plugins/org.talend.metadata.managment.ui/src/main/java/org/talend/metadata/managment/ui/model/ProjectNodeHelper.java]: SID already set in the dbInstance.fillUIConnParams(iMetadataConnection, temConnection) if not context mode

**Pull request merged**